spirv_glsl.cpp

```diff
@@ -516,18 +522,34 @@ void CompilerGLSL::emit_header()
 
 	for (auto &ext : forced_extensions)
 	{
-		if (ext == "GL_AMD_gpu_shader_half_float" && !options.vulkan_semantics)
+		if (ext == "GL_AMD_gpu_shader_half_float")
 		{
 			// Special case, this extension has a potential fallback to another vendor extension in normal GLSL.
 			// GL_AMD_gpu_shader_half_float is a superset, so try that first.
```

Comment:
wont trying a vendor-specific extension with vulkan semantics, cause the GLSL created to require a vendor-specific extension when compiling back down to SPIR-V and hence kill the vendor neutrality the original input would have had?